functions/general/fn_defactionizeType.sqf

```diff
@@ -9,8 +9,8 @@ private _defactionedClassname = "";
 } forEach [FUNC(VanillaMilitaryDefactionizer), FUNC(VanillaCivDefactionizer)];
 
 if (_defactionedClassname == "") then {
-    WARNING_1("type name of unit %1 cannot be defactionized :( defaulting to classname", _unit);
     _defactionedClassname = typeOf _unit;
+    WARNING_2("type name of unit %1 cannot be defactionized :( defaulting to classname %2", _unit, _defactionedClassname);    
```

Comment:
one `( ` too many

Reply:
even if its inside a string? preprocessor must be more stupid than i remember.

Reply:
wait no, scratch that, it's inside a string  😁